distributed_loader.cc

```diff
@@ -450,6 +451,42 @@ distributed_loader::process_upload_dir(distributed<database>& db, distributed<db
     });
 }
 
+future<std::tuple<utils::UUID, std::vector<std::vector<sstables::shared_sstable>>>>
+distributed_loader::get_sstables_from_upload_dir(distributed<database>& db, sstring ks, sstring cf) {
+    seastar::thread_attributes attr;
+    attr.sched_group = db.local().get_streaming_scheduling_group();
```

Comment:
IIRC, all REST API calls run under the streaming group anyway. So this isn't needed.

Reply:
It does not matter, it is called from storage_service::load_new_sstables which is already under streaming group. I will drop this. Thanks.